server/serverMethods.js

```diff
@@ -468,6 +470,8 @@ function ServerMethods(aLogLevel, aModules) {
           {
             userName: htmlEscape(userName || C.DEFAULT_USER_NAME),
```

Comment:
userName is declared twice

Reply:
thats going to get fixed in the linting PR :)